packages/codegen-ui-react/lib/react-component-renderer.ts

```diff
@@ -67,7 +57,7 @@ export class ReactComponentRenderer<TPropIn> extends ComponentRendererBase<
       buildOpeningElementProperties(value, key),
     );
     const eventAttributes = Object.entries(this.component.events || {}).map(([key, value]) =>
-      buildOpeningElementEvents(value, mapGenericEventToReact(key as StudioGenericEvent)),
```

Comment:
Pushed `mapGenerEventToReact` further down so that we would get the same result form `getActionIdentifier`.